Slingshot.Api/Abstract/Repository.cs

```diff
@@ -146,6 +146,11 @@ public virtual Task<bool> HasScmInfo()
             return Task.FromResult(false);
         }
 
+        public virtual Task WritePullRequestComment(string prId, string comment)
+        {
+            return Task.FromResult(new object());
```

Comment:
There is `Task.CompletedTask` in .NET 4.6. Not sure which framework this is targeting.